### PR TITLE
Merge release 2.1 branch into master branch

### PR DESCRIPTION
**Pull request by sandhyasun**

### CODE REVIEW

File: core/sqf/sqenvcom.sh [hunk 1]
```diff
@@ -32,12 +32,12 @@
 #Product version (Trafodion or derivative product)
 export TRAFODION_VER_PROD="Apache Trafodion"
 export TRAFODION_VER_MAJOR=2
-export TRAFODION_VER_MINOR=2
+export TRAFODION_VER_MINOR=1
 export TRAFODION_VER_UPDATE=0
```
Comment (svarnau) [inline core/sqf/sqenvcom.sh]: This is an error to revert release back to 2.1

Comment (sandhyasun) [inline core/sqf/sqenvcom.sh]: Oops...need to correct this. Incorrect merge.

File: install/python-installer/scripts/copy_files.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def run(user, pwd):
     hosts = dbcfgs['node_list'].split(',')
     traf_package = dbcfgs['traf_package']
 
-    sudo_prefix = get_sudo_prefix()
+    udo_prefix = get_sudo_prefix()
```
Comment (zellerh) [inline install/python-installer/scripts/copy_files.py]: Is the "s" here accidentally deleted?

Comment (sandhyasun) [inline install/python-installer/scripts/copy_files.py]: Yes ! It had to be manually resolved so got removed I guess.

**Comment by sandhyasun** [comment]
jenkins retest

**Comment by sandhyasun** [comment]
jenkins extra tests

**Comment by sandhyasun** [comment]
jenkins retest

**Pull request merged**